app/grandchallenge/challenges/models.py

```diff
@@ -982,9 +985,8 @@ def budget(self):
 
             budget["Docker storage cost"] = round(
                 self.average_algorithm_container_size_in_gb
-                * self.average_number_of_containers_per_team
```

Comment:
Since we're now removing unused container images once a day, I figured that it no longer makes sense to multiply by the `average_number_of_containers` (which is practically 1 now, and not 10 as is the default for this field).

Reply:
We still need to keep it - even though the images are being removed from ECR, we keep them on S3 in case we need to go back.

Reply:
Also - participants tend to create many algorithms and only upload one container for each, those we never remove (maybe we should - if an algorithm hasn't been used in n months we shut it down).

Reply:
Ok, and 10 is a reasonable default? The Docker storage costs then amount to 6000 Euros for the standard 50 teams, 10 containers per team of around 10GB each. Does that sound about right to you?

Reply:
From MIDOG the numbers were 48 and 314, so 50 and 5 may be good defaults.

Reply:
Ok, I can change the default value to 5 so that future requests have the updated value. I don't think it's worth writing a management command to update the existing requests. Their not really relevant anymore, and it's so few that I could also do that manually in the admin (13 type 2 requests).

Reply:
One other thing: I'm only counting the container storage costs once here, but technically they have to be counted for each phase, right?

Reply:
I think that's correct - people should not be creating new images for the subsequent phases.

Reply:
True, makes perfect sense, I just got confused between phases and challenge with multiple tasks (sorry)